src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -154,11 +154,34 @@ public String getDocStringOrNothing() {
                 getDocString().getEscapedValue() +
                 "</div></div>";
     }
+    
+    private static int labelCnt = 0;
```

Comment:
Abbreviation removed.
I put in the static to be sure that each id and for reference is unique throughout the report.